cmd/vulndash/Makefile

```diff
@@ -20,7 +20,7 @@ SHELL=/bin/bash -o pipefail
 
 REGISTRY ?= gcr.io/k8s-staging-artifact-promoter
 IMGNAME = vulndash
-IMAGE_VERSION ?= v0.3.1-1
+IMAGE_VERSION ?= v0.4.0-1
```

Comment:
I thought that as well, but I also thought this is adding a new feature, so that was the reason to bump to 0.4.

but I will move to 0.3.2, not a problem at all

Reply:
@cpanato you may be right about bump to 0.4, just wanted to bring it up

Reply:
PTAL

Reply:
We decided to stick with v0.4.0 here, though the [semver spec](https://semver.org/) leaves this a bit up to interpretation:
> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

So while patch is really only defined for major version > 0, I think it is a good policy to hold the same standard for major version = 0 patches as well.